src/ipc/dbus/ipc-dbus.cpp

```diff
@@ -54,6 +54,8 @@
 namespace facelift {
 namespace dbus {
 
+const char *const DBusIPCServiceAdapterBase::SIGNAL_TRIGGERED_SIGNAL_NAME;
```

Comment:
This has a different signature in the header. Doesn't compile: remove constexpr from header.